### PR TITLE
document class loading assumptions

### PR DESCRIPTION
**Pull request by staabm**
refs https://github.com/Roave/BackwardCompatibilityCheck/issues/216

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,6 +9,7 @@ of a PHP library.
 
  * Your project uses `git`
  * Your project uses `composer.json` to define its dependencies
+ * Composer is aware of the class loading semantics of your project, see e.g. the `"autoload"` section of `composer.json`
```
Comment (Ocramius) [inline README.md]: Not really composer, but just `composer.json`. let's rephrase this to state that all source paths are covered by an `autoload` section in `composer.json`

Comment (staabm) [reply README.md]: here we go

**Comment by Ocramius** [comment]
Failures obviously unrelated
